backend/prisma/README.md

```diff
@@ -4,3 +4,10 @@ Replace the database connection string in the schema.prisma with your own.
 
 Then uncomment out the rest of the code starting from line 10.
 
+DB Migration: https://www.prisma.io/docs/concepts/components/prisma-migrate/get-started
+- *to access the correct .env file use: dotenv -e ../.env
```

Comment:
nit: let's add something along the lines of `prefix your command with: dotnev...` since this applies to any prisma command, not just the one below

Reply:
okay i will clarify the prisma command